.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+end_of_line = lf
+indent_style = space
+trim_trailing_whitespace = true
+
+[*.java]
+indent_size = 4
+max_line_length = 100
+ij_java_block_brace_style = end_of_line
+ij_java_else_on_new_line = false
+
+[*.{txt,md}]
+max_line_length = 80
+
+[*.xml]
+indent_size = 2
```

Comment:
I'm not sure how consistent this is throughout the code base. I think the docs tend to use 2 but elsewhere it tends to be 4.